hr_timesheet_overtime/models/hr_employee.py

```diff
@@ -77,11 +77,11 @@ def _compute_current_day_working_time(self):
             employee.current_day_working_time = employee.get_working_time(current_day)
 
     @api.multi
-    @api.depends(
-        "initial_overtime",
-        "overtime_start_date",
-        "timesheet_sheet_ids.timesheet_overtime_trimmed",
-    )
+    # @api.depends(
+    #     "initial_overtime",
+    #     "overtime_start_date",
+    #     "timesheet_sheet_ids.timesheet_overtime_trimmed",
+    # )
```

Comment:
@huguesdk we spent some time on getting this right, but apparently there's a problem still.

Are we sure the reported bug is not fixed by https://github.com/coopiteasy/addons/pull/259 ? This field should _already_ be recomputed daily.

Maybe add an explicit dependency on `company_id.today` or some such?

Reply:
@carmenbianca oups I added the comment for testing, I should remove it !

Reply:
It should just work without the cron, no?

I'm not well-read on the real problem.

Reply:
Explanation from cath : En fait le problème vient du fait que le calcul ne se fait que si une modification se fait sur les timesheet donc le cron sert, en plus du calcul, à toutes les nuits forcer le calcul.

Maybe depending on company_id.today could also fix the problem.